tools/ci_build/build.py

```diff
@@ -407,6 +407,8 @@ def convert_arg_line_to_args(self, arg_line):
         help="Specify the target platform for macOS build. Only specify this argument when --build_apple_framework is present.",
     )
 
+    parser.add_argument("--visionos", action="store_true", help="build for visionOS")
```

Comment:
do we support more than one of --macos, --ios, or --visionos being given at a time?

Reply:
um good question - I think current case wouldn't involve specifying different platforms at the same time

Reply:
Let's keep it simple and only support one at a time, unless there's a concrete requirement to do otherwise.

Reply:
sounds good. in that case, it can be made clearer by making the options mutually exclusive.

https://docs.python.org/3/library/argparse.html#mutual-exclusion